CommunityFridgeMapApi/functions/fridges/v1/app.py

```diff
@@ -22,18 +22,21 @@ def lambda_handler(event: dict, ddbclient: "botocore.client.DynamoDB") -> dict:
         to the event httpMethod
         """
         httpMethod = event.get("httpMethod", None)
+        body = event.get("body", None)
         pathParameters = event["pathParameters"] or {}
         queryStringParameters = event["queryStringParameters"] or {}
         tag = queryStringParameters.get("tag", None)
         fridge_id = pathParameters.get("fridge_id", None)
+
         db_response = None
         if httpMethod == "GET":
             if fridge_id:
                 db_response = Fridge(db_client=ddbclient).get_item(fridge_id)
             else:
                 db_response = Fridge(db_client=ddbclient).get_items(tag=tag)
         elif httpMethod == "POST":
-            pass
+            body = json.loads(body)
```

Comment:
This should be in a `try-catch` construct. For an invalid `body`, this will throw an exception.

Reply:
could also do?
```
if body is not None:
    body = json.loads(body)
```

Reply:
@abhinav1912 @CollectiveFocus made that adjustment. thanks for catching it!

Reply:
@CollectiveFocus @sbeesmiley I don't think that handles the invalid JSON scenario. If `body` has malformed JSON, this would still crash.

Reply:
@abhinav1912 

okay! I will look again later and make another adjustment. thanks!

Reply:
@abhinav1912 trueee, good catch!

All of our apis are currently under the assumption that valid json format is passed in. So we could add this check to all of the apis. Not on this ticket on another one

This might be helpful @sbeesmiley https://stackoverflow.com/questions/5508509/how-do-i-check-if-a-string-is-valid-json-in-python

Reply:
I put in a try-except modeled after the example from the stackoverflow link. 
```
elif httpMethod == "POST":
            try:
                body = json.loads(body)
                db_response= Fridge(db_client=ddbclient, fridge=body).add_item()
            except ValueError as e:
                return False
```

Should I make a separate test for that case? @abhinav1912 @CollectiveFocus

Reply:
@sbeesmiley ideally, yes. We should add a test to validate if we're catching malformed strings as well.

Reply:
I created a separate ticket for this: https://trello.com/c/ep4egpeS/123-cfm115-verify-for-valid-json-on-post-requests

Need to merge this PR in early to deploy for frontend devs. Maybe @sbeesmiley you can pick it up and make a separate PR?